### PR TITLE
Mongodb connection

### PR DESCRIPTION
**Pull request by coconutperm**
Apply mongodb client connection, and default schema.

We need specific schema and connection logic for mongodb server daemon.

### CODE REVIEW

File: database/mongod.cfg [hunk 1]
```diff
@@ -4,3 +4,6 @@ systemLog:
     logAppend: true
 storage:
     dbPath: ../database
+net:
+   bindIp: 127.0.0.1
```
Comment (nadongguri) [inline database/mongod.cfg]: why bindIp and dbPath are different offset?

File: database/mongod.cfg [hunk 2]
```diff
@@ -4,3 +4,6 @@ systemLog:
     logAppend: true
 storage:
     dbPath: ../database
+net:
```
Comment (romandev) [inline database/mongod.cfg]: I think this looks unnecessary.

Please remove this parts.

File: server/db/db_connect.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const mongoose = require('mongoose');
+
+const con = mongoose.connection;
```
Comment (romandev) [inline server/db/db_connect.js]: We should use fullname for readability. (e.g. con -> connection)

File: server/db/db_connect.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+function connectServer() {
```
Comment (romandev) [inline server/db/db_connect.js]: This function should be located in common/ directory?
Also, this function can be called by any modules.

File: server/db/model/order.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
```
Comment (romandev) [inline server/db/model/order.js]: This file should be located in order module directory.

File: server/db/db_connect.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const mongoose = require('mongoose');
+
```
Comment (romandev) [inline server/db/db_connect.js]: Please remove unncessary line.

File: server/db/db_connect.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+const connect = mongoose.connection;
```
Comment (romandev) [inline server/db/db_connect.js]: This should be in connectServer()

File: server/order/model/order.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+const orderSchema = new Schema({
```
Comment (romandev) [inline server/order/model/order.js]: new mongoose.Scheme is enough.

**Pull request merged**